src/components/inspector/IssuesPanel.tsx

```diff
@@ -0,0 +1,166 @@
+/*!
+ * The contents of this file are subject to the Common Public Attribution License Version 1.0 (the “License”); you may not use this file except in compliance with the License. You may obtain a copy of the License at https://mpapp-public.gitlab.io/manuscripts-frontend/LICENSE. The License is based on the Mozilla Public License Version 1.1 but Sections 14 and 15 have been added to cover use of software over a computer network and provide for limited attribution for the Original Developer. In addition, Exhibit A has been modified to be consistent with Exhibit B.
+ *
+ * Software distributed under the License is distributed on an “AS IS” basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License for the specific language governing rights and limitations under the License.
+ *
+ * The Original Code is manuscripts-frontend.
+ *
+ * The Original Developer is the Initial Developer. The Initial Developer of the Original Code is Atypon Systems LLC.
+ *
+ * All portions of the code written by Atypon Systems LLC are Copyright (c) 2025 Atypon Systems LLC. All Rights Reserved.
+ */
+
+import { detectInconsistencyPluginKey, Warning } from '@manuscripts/body-editor'
+import { ArrowDownCircleIcon } from '@manuscripts/style-guide'
+import { NodeSelection } from 'prosemirror-state'
+import React, { useState } from 'react'
+import styled from 'styled-components'
+
+import { useStore } from '../../store'
+import { scrollIntoView } from '../track-changes/suggestion-list/Suggestion'
+
+const IssuesContainer = styled.div`
+  padding: ${(props) => props.theme.grid.unit * 2}px;
+`
+
+const CollapsibleHeader = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: ${(props) => props.theme.grid.unit * 3}px
+    ${(props) => props.theme.grid.unit * 2}px;
+  cursor: pointer;
+  margin-bottom: ${(props) => props.theme.grid.unit * 2}px;
+`
+
+const HeaderTitle = styled.h2`
+  margin: 0;
+  font-size: ${(props) => props.theme.font.size.large};
+  font-weight: 600;
+  color: ${(props) => props.theme.colors.text.primary};
+`
+
+const CollapseIcon = styled.button<{ isCollapsed: boolean }>`
+  background: #fff;
+  border: none;
+  padding: 0;
+  cursor: pointer;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  transform: ${(props) =>
+    props.isCollapsed ? 'rotate(0deg)' : 'rotate(-180deg)'};
+  transition: transform 0.2s ease;
+
+  &:hover {
+    opacity: 0.8;
+  }
+
+  &:focus {
+    outline: none;
+  }
+
+  svg {
+    width: 19px;
+    height: 19px;
+  }
+`
+
+const WarningsList = styled.div<{ isVisible: boolean }>`
+  display: ${(props) => (props.isVisible ? 'block' : 'none')};
+`
+
+const WarningItem = styled.div<{ isFocused: boolean }>`
```

Comment:
It seemed to me in the designs. That warning items is sharing the styles with a suggestion item. Can we reuse the styling from there?

Reply:
Ok, please file that ticket